src/shared/components/linkButton/linkButton.js

```diff
@@ -33,7 +35,7 @@ const LinkButton = (props) => {
 
   if (isExternalLink(link)) {
     return (
-      <a href={link} className={`${styles.linkButton} ${styles[theme]}`}>{text}</a>
+      <a href={link} target={target} rel={rel} className={`${styles.linkButton} ${styles[theme]}`}>{text}</a>
```

Comment:
Why pass it as a prop if it will always act the same manner? Just hard set target and rel.